docs/faq.rst

```diff
@@ -319,6 +319,24 @@ I'm scraping a XML document and my XPath selector doesn't return any items
 
 You may need to remove namespaces. See :ref:`removing-namespaces`.
 
+How can I split an item into multiple items in an item pipeline?
+----------------------------------------------------------------
+
+:ref:`Item pipelines <topics-item-pipeline>` cannot yield multiple items per
+input item. :ref:`Create a spider middleware <custom-spider-middleware>`
+instead, and use its
+:meth:`~scrapy.spidermiddlewares.SpiderMiddleware.process_spider_output`
+method for this puspose. For example::
+
+    class MultiplyItemsMiddleware:
+
+        def process_spider_output(self, response, result, spider):
+            for item in result:
+                if isinstance(item, scrapy.Item):
+                    for _ in range(item['multiply_by']):
+                        yield item.copy()
```

Comment:
What do you think about using `deepcopy.copy(item)` here, to avoid surprises? 
We can't use `item.deepcopy()` because item can be a dict.